### ISSUE TITLE
Console Warnings 

### ISSUE THREAD

**Issue by nicholasalanbrown** [KyleAMathews/react-headroom#21]
I think this might be due to the latest React update, as since updating I now get these warnings in my browser console:

```
Warning: ReactDOMComponent: Do not access .getDOMNode() of a DOM node; instead, use the node directly. This DOM node was rendered by `Headroom`.
```

**Comment by KyleAMathews** [comment]
Yeah I've been meaning to get to this. This would be an easy PR if you'd
like to help out.

**Comment by cbeckfield** [comment]
Added a PR for updating to React 0.14. https://github.com/KyleAMathews/react-headroom/pull/28

**Comment by perminder-klair** [comment]
Please update with PR!!!

**Comment by KyleAMathews** [comment]
Merged #28 and fixed as well the don't mutate style object warning https://github.com/KyleAMathews/react-headroom/commit/22ad662a32e47025323f7882c7ffa1d2554f41bd
